src/components/app.tsx

```diff
@@ -23,13 +23,16 @@ import { WorkspacesPage } from './pages/workspaces-page/workspaces-page';
 import { RosterColumnsPage } from './pages/roster-columns-page/roster-columns-page';
 import { SettingsPage } from './pages/settings-page/settings-page';
 import { UnitsPage } from './pages/units-page/units-page';
+import { ErrorBoundary, ErrorDialogProps } from './error-boundary/error-boundary';
 
 export const App = () => {
   const user = useSelector<AppState, UserState>(state => state.user);
   const appFrame = useSelector<AppState, AppFrameState>(state => state.appFrame);
   const dispatch = useDispatch();
   const classes = useStyles();
 
+  const [errorDialogProps, setErrorDialogProps] = useState<ErrorDialogProps>({ open: false });
```

Comment:
I think this should be in ErrorBoundary (related to my comment in the ErrorBoundary component).

Reply:
Agreed